accepted/future-releases/nnbd/feature-specification.md

```diff
@@ -424,7 +437,7 @@ the return type of the getter is not a subtype of the argument type of the
 setter.  Note that this error specifically requires subtyping and not
 assignability and hence makes no exception for `dynamic`.
 
-It is a warning to use a null aware operator (`?.`, `?..`, `??`, `??=`, or
+It is a warning to use a null aware operator (`?.`, `?[`, `?..`, `??`, `??=`, or
```

Comment:
May want to use `?[]` as the name of the operator, to not insinuate that `?[` is a single token.

Reply:
Done